### PR TITLE
More Junit Test case

### PR DESCRIPTION
**Pull request by Jiayan-Lim**
Add more Junit test case

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -448,11 +448,11 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 1. User request to add a new drug to the system
 2. The user enters drug details
    1. The following are required information:
-         - Trade name 
+         - Trade name
```
Comment (CShuwen) [inline docs/DeveloperGuide.md:451]: good removing the checkstyle error here!

File: src/test/java/seedu/careflow/model/hospital/HospitalTest.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
+package seedu.careflow.model.hospital;
```
Comment (CShuwen) [inline src/test/java/seedu/careflow/model/hospital/HospitalTest.java:1]: test cases for hospital look good to me!

File: src/test/java/seedu/careflow/model/hospital/UniqueHospitalListTest.java [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+        assertThrows(DuplicateHospitalException.class, () ->
+                uniqueHospitalList.setHospitals(listWithDuplicateHospitals));
+    }
+
+    @Test
+    public void asUnmodifiableObservableList_modifyList_throwsUnsupportedOperationException() {
+        assertThrows(UnsupportedOperationException.class, ()
+                -> uniqueHospitalList.asUnmodifiableObservableList().remove(0));
+    }
+}
```
Comment (CShuwen) [inline src/test/java/seedu/careflow/model/hospital/UniqueHospitalListTest.java:172]: did we get a checkstyle error for having no extra line at the EOF?

**Review by CShuwen — COMMENTED** [review]
LGTM, improved our test coverage significantly
May need to create a new issue and mention it in the pull request

**Comment by Jiayan-Lim** [comment]
Fix #139 
Fix #140

**Pull request merged**